_dprhtml/features/other-dialogs/init.js

```diff
@@ -5,6 +5,22 @@ class OtherDialogsViewModel {
     this.quicklinkInput = ko.observable();
     this.quicklinkInNewTab = ko.observable(false);
     this.bookmarkName = ko.observable();
+    this.downloadTranslation("bt");
+  }
+
+  async downloadTranslation(source) {
+    await DPR_PAL.addOneJS(`translations_${source}_list`);
```

Comment:
this file needs to be outside _dprhtml, otherwise it will get included in the shell in sw.js automatically by workbox command line. we want to keep the shell as light weight as possible to ensure install/startup time is quick.

i recommend /translations/bt/translations_list.js